react-ui/src/components/Layouts/InputWorkerTimes/WorkerInputpage.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import Check from './CheckboxApplication';
+import './WorkerInputpage.scss';
+import { Link } from 'react-router-dom';
+import { Container, Row, Col } from 'react-grid-system';
+import { Redirect, Route, withRouter } from "react-router-dom";
+import ReactDOM from "react-dom";
+import moment from 'moment';
+
```

Comment:
Hi Kyle,
If possible, I would recommend removing the whitespaces on lines 10 to 13 to reduce unnecessarily long files.

Reply:
Done